test/suites/integration/subnet_test.go

```diff
@@ -108,3 +131,35 @@ func getSubnets(tags map[string]string) map[string][]string {
 	Expect(err).To(BeNil())
 	return subnets
 }
+
+// SubnetInfo is a simple struct for testing
+type SubnetInfo struct {
```

Comment:
Consider just passing the ec2 subnets around, rather than making another struct.